.github/workflows/docker-publish.yml

```diff
@@ -22,18 +22,18 @@ on:
       #     - nginx
       #     - apache
       ps_version:
-        description: "The target PrestaShop version"
-        required: true
+        description: "The target PrestaShop version (defaults to 'latest')"
+        required: false
         type: string
 
       php_version:
-        description: "The target PHP version"
+        description: "The target PHP version (defaults to 'auto')"
```

Comment:
```suggestion
        description: "The target PHP version (defaults to 'auto')"
        default: auto
```

Reply:
"auto" isn't yet managed within the `build.sh` file. Won't fix this now, but later on this could provide better understandability for our users